app/controllers/rooms_controller.rb

```diff
@@ -346,6 +352,22 @@ def home
 
   def controller
   end
+
+  def leaderboard
+    season_id = @room.get_active_season.id
```

Comment:
My only thought here is that this will error if we have no seasons. But, if that's happening then we just have to make one!